lmdeploy/cli/utils.py

```diff
@@ -342,7 +352,8 @@ def calib_search_scale(parser):
 
         return parser.add_argument(
             '--search-scale',
-            type=bool,
+            action='store_true',
```

Comment:
Is `search-scale` time consuming?

Reply:
Yes, it is time-consuming. By default, it is False.